build-system/common/ci.js

```diff
@@ -194,6 +194,15 @@ function ciJobUrl() {
     : '';
 }
 
+/**
+ * Returns the merge commits SHA when running a CI Pull Request build.
+ * @return {string}
+ */
+function ciMergeSha() {
+  // CIRCLE_MERGE_SHA is populated by .circleci/fetch_merge_commit.sh.
+  return isCircleci ? env('CIRCLE_MERGE_SHA') : '';
```

Comment:
Should this condition be `isPullRequestBuild() && isCircleciBuild()` to guarantee an empty value during push builds?